tunder/lib/src/database/operations/contracts/count_operation.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:tunder/extensions.dart';
+import 'package:tunder/database.dart';
+import 'package:tunder/src/database/operations/postgres/postgres_count_operation.dart';
+
+abstract class CountOperation {
+  factory CountOperation.forDatabase(Symbol driver) {
```

Comment:
I'd either rename the factory constructor to "forDriver" or rename the Symbol parameter to "database" to improve the naming consistency.

Reply:
Agreed! I'll do that. 👍🏻